### ISSUE TITLE
Usage on OS X + arm64

### ISSUE THREAD

**Issue by coding-velociraptor** [microsoft/node-api-dotnet#211]
Hey team,

Thank you for providing this library aiming to improve developer experience during NodeJS and C# interopability.

I'm trying to run it locally without success. For reproduction, I created this very [simple repository](https://github.com/coding-velociraptor/node-api-dotnet-simple). When I run the project `node main.js`, I get the following error message:

```bash
Unhandled Exception: System.ArgumentNullException: A runtime is required. Arg_ParamName_Name, runtime
   at Microsoft.JavaScript.NodeApi.JSValueScope..ctor(JSValueScopeType, JSRuntime.napi_env, JSRuntime, JSSynchronizationContext) + 0x450
   at Microsoft.JavaScript.NodeApi.JSValue.InvokeCallback[TDescriptor](JSRuntime.napi_env, JSRuntime.napi_callback_info, JSValueScopeType, Func`2) + 0x7c
   at Microsoft.JavaScript.NodeApi.JSValue.InvokeJSGetterNoContext(JSRuntime.napi_env, JSRuntime.napi_callback_info) + 0x70
[1]    70557 abort      node main.js
```

I tried `node-api-dotnet/net8.0` and `node-api-dotnet/net6.0`, installing dotnet v6 and also "NODE_API_TRACE_HOST" like described in [this issue](https://github.com/microsoft/node-api-dotnet/issues/203)

Here are some system information: 

```bash
dotnet --version
> 8.0.201

# Installed using nvm.sh
node --version
> v18.19.0
```

Proc: Apple M1 Pro 
OS: Sonoma 14.2.1

How might I fix this issue?

**Comment by jasongin** [comment]
@vmoroz Was this fixed by #222 ?

**Comment by jasongin** [comment]
@coding-velociraptor This is fixed in v0.5.10.

**Comment by vmoroz** [comment]
@coding-velociraptor , thank you for reporting the issue and for the repro simple repository!
It helped a lot to investigate the issue and to verify the fix.
